src/components/PostProcessing/LinkContent.tsx

```diff
@@ -1,24 +1,20 @@
-import Arrow from 'icons/Arrow'
-import classnames, {
-  alignItems,
-  display,
-  gap,
-  width,
-} from 'classnames/tailwind'
+import Button from 'components/Button'
 
-const linkInnerContainer = classnames(
-  display('flex'),
-  alignItems('items-center'),
-  gap('gap-x-2')
-)
-
-export default function ({ text }: { text: string }) {
+export default function ({
+  text,
+  onClick,
+}: {
+  text: string
+  onClick?: () => void
+}) {
   return (
-    <div className={linkInnerContainer}>
-      <span>{text}</span>
-      <div className={width('w-4')}>
-        <Arrow horizontal />
-      </div>
-    </div>
+    <Button
+      small
+      gradientFont
+      withArrow
+      onClick={() => (onClick ? onClick() : undefined)}
```

Comment:
Why not make `onClick` not an optional argument?

Reply:
Then it will be on the component upper

Reply:
I don't think it's needed, because of comment above